lms/djangoapps/etc_api/views.py

```diff
@@ -44,6 +45,7 @@ def post(self, request):
         password = request.data.get('password')
         prename = request.data.get('prename', '')
         surname = request.data.get('surname', '')
+        is_send_email = not bool(password)
```

Comment:
Please add a comment

Reply:
I added the comment to the code. 
`
If the password wasn't sent we would generate a password and send the email for changing it.
`